22:46:12,731 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
22:46:12,768 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
22:46:12,778 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
22:46:12,787 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-33 to job manager
22:46:12,852 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
22:46:12,852 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
22:46:12,853 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
22:46:13,948 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
22:46:13,966 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
22:46:13,976 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
22:46:13,976 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
22:46:46,189 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (13/25)
22:46:46,193 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (13/25)
22:46:47,465 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (101/200)
22:46:47,465 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (101/200)
22:46:47,467 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (101/200)
22:46:47,468 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (101/200)
22:46:48,833 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (103/200)
22:46:48,834 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (103/200)
22:46:48,834 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (103/200)
22:46:48,835 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (103/200)
22:46:50,221 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (99/200)
22:46:50,222 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (99/200)
22:46:50,223 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (99/200)
22:46:50,224 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (99/200)
22:46:50,525 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (100/200)
22:46:50,525 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (100/200)
22:46:50,526 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (100/200)
22:46:50,527 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (100/200)
22:46:50,931 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (97/200)
22:46:50,932 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (97/200)
22:46:50,933 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (97/200)
22:46:50,934 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (97/200)
22:46:51,318 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (104/200)
22:46:51,318 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (104/200)
22:46:51,320 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (104/200)
22:46:51,321 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (104/200)
22:46:51,992 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (98/200)
22:46:51,992 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (98/200)
22:46:51,994 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (98/200)
22:46:51,995 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (98/200)
22:46:53,035 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (102/200)
22:46:53,035 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (102/200)
22:46:53,037 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (102/200)
22:46:53,038 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (102/200)
22:47:21,053 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (13/25)
22:47:21,053 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (13/25)
22:49:29,695 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7226217950160610329_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:53624 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:29,696 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7226217950160610329_1722 bad datanode[0] 130.149.21.63:45020
22:49:29,697 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7226217950160610329_1722 in pipeline 130.149.21.63:45020, 130.149.21.16:45020, 130.149.21.33:45020: bad datanode 130.149.21.63:45020
22:49:32,984 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_718306188316257863_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:53622 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:32,985 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_718306188316257863_1722 bad datanode[0] 130.149.21.63:45020
22:49:32,985 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_718306188316257863_1722 in pipeline 130.149.21.63:45020, 130.149.21.22:45020, 130.149.21.25:45020: bad datanode 130.149.21.63:45020
22:49:59,376 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-335794325459332931_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:53620 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:59,377 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-335794325459332931_1722 bad datanode[0] 130.149.21.63:45020
22:49:59,377 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-335794325459332931_1722 in pipeline 130.149.21.63:45020, 130.149.21.66:45020, 130.149.21.2:45020: bad datanode 130.149.21.63:45020
22:50:05,767 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_1883429874375403422_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:53633 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:05,768 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1883429874375403422_1722 bad datanode[0] 130.149.21.63:45020
22:50:05,769 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1883429874375403422_1722 in pipeline 130.149.21.63:45020, 130.149.21.61:45020, 130.149.21.29:45020: bad datanode 130.149.21.63:45020
22:50:07,919 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_4431844978244067608_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:53635 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:07,920 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4431844978244067608_1722 bad datanode[0] 130.149.21.63:45020
22:50:07,921 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4431844978244067608_1722 in pipeline 130.149.21.63:45020, 130.149.21.17:45020, 130.149.21.66:45020: bad datanode 130.149.21.63:45020
22:50:08,265 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_8704804919093834527_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:53613 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:08,266 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8704804919093834527_1722 bad datanode[0] 130.149.21.63:45020
22:50:08,267 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8704804919093834527_1722 in pipeline 130.149.21.63:45020, 130.149.21.60:45020, 130.149.21.65:45020: bad datanode 130.149.21.63:45020
22:50:08,487 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_211713987818197978_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:53618 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:08,488 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_211713987818197978_1722 bad datanode[0] 130.149.21.63:45020
22:50:08,489 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_211713987818197978_1722 in pipeline 130.149.21.63:45020, 130.149.21.16:45020, 130.149.21.25:45020: bad datanode 130.149.21.63:45020
22:50:10,256 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7591235464249216275_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:53626 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:10,257 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7591235464249216275_1722 bad datanode[0] 130.149.21.63:45020
22:50:10,257 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7591235464249216275_1722 in pipeline 130.149.21.63:45020, 130.149.21.60:45020, 130.149.21.23:45020: bad datanode 130.149.21.63:45020
22:50:35,808 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7226217950160610329_1768java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:34979 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:35,809 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7226217950160610329_1768 bad datanode[0] 130.149.21.16:45020
22:50:35,809 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7226217950160610329_1768 in pipeline 130.149.21.16:45020, 130.149.21.33:45020: bad datanode 130.149.21.16:45020
22:50:39,086 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_718306188316257863_1779java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:45017 remote=/130.149.21.22:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:39,087 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_718306188316257863_1779 bad datanode[0] 130.149.21.22:45020
22:50:39,088 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_718306188316257863_1779 in pipeline 130.149.21.22:45020, 130.149.21.25:45020: bad datanode 130.149.21.22:45020
22:51:05,482 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-335794325459332931_1822java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:44168 remote=/130.149.21.66:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:05,483 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-335794325459332931_1822 bad datanode[0] 130.149.21.66:45020
22:51:05,484 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-335794325459332931_1822 in pipeline 130.149.21.66:45020, 130.149.21.2:45020: bad datanode 130.149.21.66:45020
22:51:11,868 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_1883429874375403422_1836java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:55060 remote=/130.149.21.61:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:11,869 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1883429874375403422_1836 bad datanode[0] 130.149.21.61:45020
22:51:11,870 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1883429874375403422_1836 in pipeline 130.149.21.61:45020, 130.149.21.29:45020: bad datanode 130.149.21.61:45020
22:51:14,016 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_4431844978244067608_1844java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:35513 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:14,017 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4431844978244067608_1844 bad datanode[0] 130.149.21.17:45020
22:51:14,018 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4431844978244067608_1844 in pipeline 130.149.21.17:45020, 130.149.21.66:45020: bad datanode 130.149.21.17:45020
22:51:14,365 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_8704804919093834527_1845java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:32835 remote=/130.149.21.60:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:14,366 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8704804919093834527_1845 bad datanode[0] 130.149.21.60:45020
22:51:14,366 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8704804919093834527_1845 in pipeline 130.149.21.60:45020, 130.149.21.65:45020: bad datanode 130.149.21.60:45020
22:51:14,586 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_211713987818197978_1846java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:34994 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:14,587 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_211713987818197978_1846 bad datanode[0] 130.149.21.16:45020
22:51:14,588 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_211713987818197978_1846 in pipeline 130.149.21.16:45020, 130.149.21.25:45020: bad datanode 130.149.21.16:45020
22:51:16,354 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7591235464249216275_1854java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.63:32839 remote=/130.149.21.60:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:16,356 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7591235464249216275_1854 bad datanode[0] 130.149.21.60:45020
22:51:16,356 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7591235464249216275_1854 in pipeline 130.149.21.60:45020, 130.149.21.23:45020: bad datanode 130.149.21.60:45020
